Formula/trino.rb

```diff
@@ -1,15 +1,20 @@
 class Trino < Formula
   desc "Distributed SQL query engine for big data"
   homepage "https://trino.io"
-  url "https://search.maven.org/remotecontent?filepath=io/prestosql/presto-server/344/presto-server-344.tar.gz"
-  sha256 "9ae950f2901efd5cb1ca7d1bbd8a4cbb01d16dfe9c4fe702db2ee147ab841a8b"
+  url "https://search.maven.org/remotecontent?filepath=io/trino/trino-server/360/trino-server-360.tar.gz"
+  sha256 "cd313867fd5461a26920ca6cb170413893853b0cc1acc4f454d4141a0790a50e"
   license "Apache-2.0"
 
+  livecheck do
+    url "https://search.maven.org/remotecontent?filepath=io/trino/trino-server/"
+    regex(%r{href=["']?v?(\d+(?:\.\d+)+)/?["' >]}i)
+  end
```

Comment:
i copied this from `prestodb`: https://github.com/Homebrew/homebrew-core/blob/c2a2a9782abc3b39e5bc70d3bfeaffdaba73148d/Formula/prestodb.rb#L11-L14

however, it isn't working for me. would appreciate any pointers here:
```
$ brew livecheck trino
Error: trino: Unable to get versions
```